packages/shared/lib/core/utils/unit.ts

```diff
@@ -24,20 +20,22 @@ export function convertIotaUnit(value: number, fromUnit: IotaUnit, toUnit: IotaU
     return toUnit === IotaUnit._ ? Math.round(scaledValue) : scaledValue
 }
 
-export function formatIotaUnitPrecision(
-    valueRaw: number,
-    unit: IotaUnit,
-    includeUnits: boolean = true,
-    grouped: boolean = false,
+interface FormatIotaUnitOptions {
+    includeUnits?: boolean
+    grouped?: boolean
     overrideDecimalPlaces?: number
-): string {
+}
```

Comment:
We can put this in the util's `interfaces` directory so it's accessible outside of this file when we use this function